modules/fedimint-wallet-server/src/lib.rs

```diff
@@ -71,6 +75,58 @@ use secp256k1::{Message, Scalar};
 use strum::IntoEnumIterator;
 use tracing::{debug, error, info, instrument, trace, warn};
 
+lazy_static! {
+    static ref WALLET_PEGIN: IntCounter =
+        register_int_counter!(opts!("wallet_pegin", "Wallet peg-in")).unwrap();
```

Comment:
+ `_total`?

https://prometheus.io/docs/practices/naming/

Reply:
I'll just remove this